eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Internal.hs

```diff
@@ -266,11 +266,20 @@ instance EraPParams era => NoThunks (EnactState era)
 
 -- ========================================
 
+-- | `RatifyState` stores information about what will happen to the active
+-- governance actions at the next epoch boundary.
 data RatifyState era = RatifyState
   { rsEnactState :: !(EnactState era)
```

Comment:
Could you also add documentation on what this `EnactState` actually means?
Something along the lines of "This state is the result of running `ENACT` rule. It will contain all the changes that resulted from enactment of all proposal next epoch if any"

Reply:
Done